solana-programs/README.md

```diff
@@ -92,6 +92,8 @@ Then, run:
 ./scripts/init-dev-fixtures.sh
 ```
 
+NOTE: The Reward Manager Lookup Table is not included in this script. To recreate the Reward Manager lookup table, see packages/libs/src/sdk/scripts/manageRewardsLookupTable.ts. You'll then have to export it to a file using the same commands found in the init-dev-fixtures.sh script
```

Comment:
could we get an example of how to call the script somewhere? would be useful to see examples of both create and update.